content/guides/asset-management.md

```diff
@@ -4,163 +4,522 @@ sort: 3
+   [2] ./~/css-loader!./src/style.css 191 bytes {0} [built]
+   [3] ./~/css-loader/lib/css-base.js 2.26 kB {0} [built]
+   [4] ./~/style-loader/lib/addStyles.js 8.7 kB {0} [built]
+   [5] ./~/style-loader/lib/urls.js 3.01 kB {0} [built]
+   [6] (webpack)/buildin/global.js 509 bytes {0} [built]
+   [7] (webpack)/buildin/module.js 517 bytes {0} [built]
+   [8] ./src/index.js 351 bytes {0} [built]
+```
+
+Open up `index.html` in your browser again and you should see that `Hello webpack` is now styled in red. If you want to see what webpack did, inspect the page (don't view the page source, as it won't show you the result) and look at the page's head tags. It should contain our style block that we imported in `index.js`:
+
+``` html
+<html>
+  <head>
+    <title>Asset Management</title>
+    <style type="text/css">
+    .hello {
+      color: red;
+    }
+    </style>
+  </head>
+  <body>
+    <script src="./bundle.js"></script>
+    <div class="hello">Hello webpack</div>
+  </body>
+</html>
+```
+
 T> Note that you can also [split your CSS](/guides/code-splitting-css) for better load times in production. On top of that, loaders exist for pretty much any flavor of CSS you can think of -- [postcss](/loaders/postcss-loader), [sass](/loaders/sass-loader), and [less](/loaders/less-loader) to name a few.
 
 
 ## Loading Images
 
-So now we're pulling in our CSS, but what about our backgrounds and icons? Using the [file-loader](/loaders/file-loader) we can easily incorporate those in our system as well:
+So now we're pulling in our CSS, but what about our images like backgrounds and icons? Using the [file-loader](/loaders/file-loader) we can easily incorporate those in our system as well:
 
 ``` bash
 npm install --save-dev file-loader
 ```
 
 __webpack.config.js__
 
-``` js
+``` diff
+var path = require('path');
+
 module.exports = {
-  //...
+  entry: './src/index.js',
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, 'dist')
+  },
   module: {
     rules: [
       {
-        test: /\.(png|svg|jpg|gif)$/,
+        test: /\.css$/,
         use: [
-          'file-loader'
-        ]
+          'style-loader',
+          'css-loader'
+        ]
       }
++      {
++        test: /\.(png|svg|jpg|gif)$/,
++        use: [
++          'file-loader'
++        ]
++      }
     ]
-  },
-  //...
+  }
+};
+
+```
+
+Now, when you `import someImage from './my-image.png'`, that image will be processed and added to your `output` directory _and_ the `someImage` variable will contain the final url of that image after processing. When using the [css-loader](/loaders/css-loader), as shown above, a similar process will occur for `url('./my-image.png')` within your CSS. The loader will recognize this is a local file, and replace the `'./my-image.png'` path with the final path to the image in your `output` directory. The [html-loader](/loaders/html-loader) handles `<img src="./my-image.png" />` in the same manner.
```

Comment:
Yup, this will result in an error: `Uncaught TypeError: __WEBPACK_IMPORTED_MODULE_0__image_png___default.a is not a constructor` if you assign it to `Image` and then later call `var i = new Image();`, so let's change it.